src/ProjectService.Business/Commands/Task/FindTasksCommand.cs

```diff
@@ -126,7 +149,7 @@ public FindResultResponse<TaskInfo> Execute(FindTasksFilter filter)
       if (!_accessValidator.IsAdmin()
```

Comment:
you can remove isAdmin check, you have right chesk

Reply:
fixed